### PR TITLE
Readme.md/adding images and animation

### PR DESCRIPTION
**Pull request by patrycja-gomza**
Hi Team 👋

I've made some visual enhancements to our README. Here's a summary of the changes:

1. Add PNG and GIF Files
2. Implement Image and Animation in README
3. Delete Outdated README Image

Feel free to review the changes, and if everything looks good, please approve the pull request. Thanks! 😊

### CODE REVIEW

File: readme_gifAnimation.gif
[the remark this answers is not in the document]
Comment (patrycja-gomza) [reply readme_gifAnimation.gif:1]: Hi @bedicooper 👋
I appreciate your feedback, and I apologize for overlooking the size of the .gif file. I'll work on reducing its size to improve loading times.

[the remark this answers is not in the document]
Comment (bedicooper) [reply readme_gifAnimation.gif:1]: 👍

**Review by bedicooper — APPROVED** [review]
Great! smaller gif size will work much bettter. I was able to watch it now, it nicely represents our app. 👏

**Comment by patrycja-gomza** [comment]
> Great! smaller gif size will work much bettter. I was able to watch it now, it nicely represents our app. 👏

Thank you 😊

**Pull request merged**